CHANGELOG.md

```diff
@@ -2,6 +2,96 @@
 
 This document outlines major changes between releases.
 
+## 0.107.0 "Multithreading/Multiplication" (02 Dec 2024)
+
+A large update that except for numerous enhancements and bug fixes introduces a
+major node extension: NeoFS BlockFetcher service and `util upload-bin` CLI command
+implemented as a part of [NeoFS snapshot storage
+proposal](https://github.com/neo-project/neo/issues/3463). BlockFetcher service, as
+an alternative to P2P synchronisation mechanism, allows to download blocks and sync
+chain from block dumps stored in NeoFS. Starting from this release, NeoSPCC team
+maintains chain dumps in NeoFS for N3 and NeoFS public networks. Default NeoGo node
+configuration for these networks has been changed to use NeoFS BlockFetcher as a
+default synchronisation start point prior to P2P synchronisation. Other than that,
+this release includes NEP-24 standard support at both compiler and SC bindings
+generator levels. A large number of tiny user-facing enhancements is rolled out for
+RPC actor/ivoker, `neotest` and `unwrap` packages as far as for CLI utilities.
+Also, this release contains a set of NeoGo VM bug fixes inspired by differential
+VM fuzzing study conducted by our external contributor @Slava0135 and a set of tiny
+VM CLI enhancements introduced by @ixje.
+
+Some deprecated functionality has been removed according to the schedule, see more
+details in the `Behaviour changes` section. Also, for those node operators who would
+like to check out our new NeoFS BlockFetcher node extension, we'd recommend to add
+corresponding section to the node's configuration (see the default node configuration
+for N3/NeoFS networks for example).
+
+This release is fully compatible with 3.7.5 version of C# node, no DB resync is
+needed.
+
+New features:
+ * new NeoFS BlockFetcher service that allows to sync node from NeoFS chain dump
+   (#3515, #3636, #3637, #3632, #3691, #3706, #3668, #3713)
+ * new `util upload-bin` CLI command that allows to upload blocks from Neo chains to
+   NeoFS (#3578, #3625, #3626, #3633, #3637, #3638, #3643, #3650, #3662, #3684,
+   #3686, #3691)
+ * new `delete` and `ib` VM CLI commands for brealpoints management (#3674)
+ * NEP-24 standard support (#3560)
+ * Echidna hardfork introduced, but not yet enabled (#3554)
```

Comment:
Maybe signify that it's a Echidna preview, it's just a part of changes, it'll change in an incompatible way before the full support and can be enabled for experiments only.